contracts/deploy/mainnet/095_ogn_buyback.js

```diff
@@ -45,7 +46,8 @@ module.exports = deploymentWithGovernanceProposal(
       true
     );
 
-    await cSwapper.approveAssets([addresses.mainnet.OGN]);
+    // TODO: resolve this at some point
+    //await cSwapper.approveAssets([addresses.mainnet.OGN]);
```

Comment:
FIxing the CI on the master. @shahthepro is it ok to comment the above couple of lines out for now?

Reply:
Yep, it should be fine